src/js/components/initialmessage.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @param {object} data contains the expected page content data
+ * returns a default component to show the provided information.
+ */
+const initialMessageComp = (data) => {
+  const section = document.createElement('SECTION');
+  
+  if (data.backgroundColor) section.classList.add(`textblockcomp__container--${data.backgroundColor}`);
```

Comment:
Please use - to separate compound words in classes

'text-block-comp'

Reply:
Done!!!